network-api/networkapi/views.py

```diff
@@ -19,3 +22,15 @@ def review_app_help_view(request):
         return render(request, "reviewapp-help.html")
     else:
         return HttpResponse(status=404)
+
+
+@require_GET
+def apple_pay_domain_association_view(request):
+    """
+    Returns string needed for Apple Pay domain association/verification.
+    """
+    if Site.find_for_request(request).hostname == "www.mozillafestival.org":
```

Comment:
Hi @mtdenton!

Had a quick question about this line and one of the dev tasks in the ticket:

Dev Task: "Create two **_production_** environment variables on Heroku, one for each file provided, and store the file's contents. Keep the names mnemonic for their respective base URLs"

Based on the dev task above, I am only checking for the production URL of Mozfest. I was wondering however if we wanted to instead update this to include the staging URL (www.mozillafestival.mofostagin.net) and the local dev URL (mozfest.localhost:8000) for testing purposes?

Or, is apple pay verification something that only happens on prod?

Thanks in advance!

Reply:
@danielfmiranda I think hardcoding host names is not a good idea, exactly because of the different environments. It would also require code changes when the configuration changes (e.g. the domain). 

The foundation site and the Mozfest site have separate page types for their homepage. You could use that to determine which key should be served.

Reply:
Great suggestion, updated!